bab/1-pendahuluan.tex

```diff
@@ -54,30 +54,30 @@ \section{Sistematika Penulisan}
 
   \item \textbf{BAB I Pendahuluan}
 
-  Bab ini berisi \lipsum[2][1-5]
+        Bab ini berisi \lipsum[2][1-5]
```

Comment:
do we really need indentation here?

Reply:
im using latex workshop formatter, it formats just like this

Reply:
Okay then, later let's add another PR to format the whole files using that formatter.